src/main/osd/osd_elements.c

```diff
@@ -783,20 +783,25 @@ static void osdBackgroundCameraFrame(osdElementParms_t *element)
     element->drawElement = false;  // element already drawn
 }
 
+static void toUpperCase(char* dest, const char* src, unsigned maxSrcLength)
```

Comment:
```suggestion
static void toUpperCase(const char *src, char *dest, unsigned maxSrcLength)
```

I'd propose to make parameter `src` go first in the list, so that we know that `toUpperCase` is meant to be applied to `src` and not `dest`. (It's common practice to apply a function to the first parameter, then list everything else afterwards)

Reply:
I was always thinking dest goes first in C
For example, check classic memcpu
`void *memcpy(void *dest, const void *src, size_t n)`

Reply:
Ah, I see. That's why you put it this way around. Decide for yourself. I'm also fine with the way it is right now. I was just wondering because it's not BF convention. But it makes sense how you did it as well.

Reply:
To me dest first was always weird in C/C++ lol
But here i just searched through BF by "dest" it shows:
`return max7456LayerCopy(destLayer, sourceLayer)`
`void * memcpy_fn ( void * destination, const void * source, size_t num ) asm("memcpy");`
`applyMatrixRotation(dest, sensorRotationMatrix);`
`static void memory_read_proc(uint8_t *dest, int size, uint32_t offset, emfat_entry_t *entry)`

but also there are the opposite
`static void cRleEncodeStream(sbuf_t *source, sbuf_t *dest, uint8_t maxDestLen)`
`int scaleRange(int x, int srcMin, int srcMax, int destMin, int destMax)`
so who knows lol

Reply:
I guess you could argue that `toUpperCase()` is applied to `dest` and also manipulates `dest`, so if you look at it from this perspective it makes sense again to make it the first parameter.

Reply:
See https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#parameter-order